Stripe/STPAPIClient.m

```diff
@@ -336,8 +334,9 @@ @implementation STPAPIClient (CreditCards)
 
 - (void)createTokenWithCard:(STPCard *)card completion:(STPTokenCompletionBlock)completion {
     NSMutableDictionary *params = [[STPFormEncoder dictionaryForObject:card] mutableCopy];
-    params[@"muid"] = [STPAnalyticsClient muid];
+    [[STPTelemetryClient sharedInstance] addTelemetryFieldsToParams:params];
     [self createTokenWithParameters:params completion:completion];
```

Comment:
ah yeah, this is intentional – we're starting with only card tokens, but we'll be passing the same stuff to source creations in the future.